contracts/tokens/Cruzo1155.sol

```diff
@@ -147,4 +153,36 @@ contract Cruzo1155 is Initializable, ERC1155URI {
     function setContractURI(string memory _newURI) external onlyOwner {
         contractURI = _newURI;
     }
+    function supportsInterface(bytes4 interfaceId)
+        public
+        view
+        override(ERC2981Upgradeable, ERC1155Upgradeable)
+        returns (bool)
+    {
+        return super.supportsInterface(interfaceId);
+    }
+
+    function setDefaultRoyaltyInfo(address _receiver, uint96 _royaltyFeesInBips)
+        public
+        onlyOwner
+    {
+        require(
```

Comment:
@suNrisEinMyeYes Would it make sense to introduce a modifier with this `require`?

Reply:
Can be done later with low priority.